src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,67 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (newPage: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const linksAmount = Math.ceil(total / perPage);
```

Comment:
Make sure to handle the edge case where `perPage` is 0 to avoid division by zero which will result in `linksAmount` being `Infinity`.